packages/next-app/src/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/index.tsx

```diff
@@ -123,14 +123,29 @@ function Row({
           </div>
         )}
       </Td>
-      <Td className="min-w-[10rem]">{data.key}</Td>
+      <Td className="min-w-[5rem] max-w-[10rem]">
+        {!data.value || !data.description ? (
+          <div className="flex justify-between">
+            <>{data.key}</>
+            <span className="inline-flex items-center rounded-md bg-red-200 p-0.5 text-xs font-semibold text-red-600">
+              NEEDS EDIT
+            </span>
+          </div>
+        ) : (
+          <>{data.key}</>
+        )}
```

Comment:
perhaps this could be 
```suggestion
          <div className="flex justify-between">
            <>{data.key}</>
            {!data.value || !data.description && (
                  <span className="inline-flex items-center rounded-md bg-red-200 p-0.5 text-xs font-semibold text-red-600">
                    NEEDS EDIT
                  </span>
                </div>
            )}
```